app-text/booksorg/booksorg-0.3.recipe

```diff
@@ -0,0 +1,84 @@
+SUMMARY="A Simple Books Organizer"
+DESCRIPTION="Application to Organize with ease your pdf book"
+COPYRIGHT="2014-2015 Abou Zakaria"
+LICENSE="GNU GPL v3"
+REVISION="1"
+HOMEPAGE="https://sourceforge.net/projects/booksorg/"
+SOURCE_URI="https://downloads.sourceforge.net/booksorg/booksorg_${portVersion/./}.tar.gz"
+SOURCE_DIR="booksorg_${portVersion/./}"
+CHECKSUM_SHA256="dcdcce89090d1539c020412d294e2f6c0ffc84a0f4f400132f57e1c1be29d234"
+PATCHES="booksorg-$portVersion.patchset"
+ADDITIONAL_FILES="booksorg.rdef.in"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+SECONDARY_ARCHITECTURES="x86"
+fi
+
+USER_SETTINGS_FILES="
+	settings/elokab/BooksOrg directory
+	"
```

Comment:
```suggestion
	"

```

Per the [guidelines](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering)

Reply:
@fbrosson you forgot this one :P